rest_framework_simplejwt/backends.py

```diff
@@ -5,14 +5,17 @@
 from .exceptions import TokenBackendError
 from .utils import format_lazy
 
-ALLOWED_ALGORITHMS = (
+ALLOWED_ALGORITHMS = {
     "HS256",
     "HS384",
     "HS512",
     "RS256",
     "RS384",
     "RS512",
-)
+    "ES256",
+    "ES384",
+    "ES512",
+}
```

Comment:
please leave this as a tuple

Reply:
there doesn't seem to be any problem with using set. what do you think?

Reply:
backwards compatibility